### DISCUSSION TITLE
Why are the panels size so small ? How can I zoom in ?

### DISCUSSION THREAD

**Discussion by aculnaig** [WerWolv/ImHex#1800] [Feedback]
![Screenshot from 2024-07-06 08-44-42](https://github.com/WerWolv/ImHex/assets/9980529/8cfabe95-8d97-4da5-84bd-c12cbc5741c2)

**Comment by aculnaig** [comment]
```text
cat /etc/lsb-release 
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=22.04
DISTRIB_CODENAME=jammy
DISTRIB_DESCRIPTION="Ubuntu 22.04.4 LTS"
```

**Reply by WerWolv** [reply]
Hey, you're probably using a really high resolution screen. Go to Extras -> Settings -> Interface and increase the scaling there to maybe 2x or 3x. That should make it look better

**Reply by aculnaig** [reply]
Thank you very much ! Now I can see perfectly fine !
